docs: minor fixes to transaction signing sample code and content (#830)
blockstack/docs: src/pages/smart-contracts/signing-transactions.md

// Here's an example of options:
const myStatus = 'hey there';

> const opts = {

Reviewer: I prefer the verbose `options` vs `opts`. May I ask why you changed it? (if it's because that's the interface name in connect, perhaps we should change that to `options` in place of `opts`)

Author: I changed it because the sample code doesn't work.. further down in the code, `opts` is used as an input for the method call

Reviewer 2: Yes, I would also prefer `options`, and then fixing the use of `opts` to be `options` lower in the doc.

Author: cool, done

await openContractCall(opts);
```

> When calling `openContractCall`, you need to specify a few details. Here is the exact interface that describes what options you have:

Reviewer: ```suggestion
There are some required parameters needed when calling `openContractCall`. Here is the exact interface which describes what options you have:
```
